config/routes.rb

```diff
@@ -5,6 +5,11 @@
   root 'users#index'
 
   resources :users, only: [:index, :show] do
-    resources :posts, only: [:index, :show]
+    resources :posts, only: [:index, :show] do
+      post 'like', to: 'likes#create'
+      resources :comments, only: [:create]
+    end
   end
+
+  resources :posts, only: [:new, :create]
```

Comment:
- [ ][OPTIONAL] the new post routes has to still belong to a user so it is suppose to be users/1/posts/new not posts/new